firedrake/mesh.py

```diff
@@ -1522,6 +1584,16 @@ def __dir__(self):
         current = super(MeshGeometry, self).__dir__()
         return list(OrderedDict.fromkeys(dir(self._topology) + current))
 
+    def view(self, fname, formt):
```

Comment:
Rename as save. Also needs a doctoring. Please also spell out filename and format.